xml/System/DateOnly.xml

```diff
@@ -150,6 +150,9 @@
         <param name="value">The number of days to add. To subtract days, specify a negative number.</param>
         <summary>Adds the specified number of days to the value of this instance.</summary>
         <returns>An instance whose value is the sum of the date represented by this instance and the number of days represented by value.</returns>
+        <exception cref="T:System.ArgumentOutOfRangeException">
+          Thrown if the resulting value would be greater than <see cref="MaxValue"/>.
```

Comment:
><see cref="MaxValue"/> [](http://example.com/codeflow?start=62&length=22)

<see cref="F:System.DateOnly.MaxValue">DateOnly.MaxValue</see>

CC @geneviven  to confirm.